### ISSUE TITLE
[BUG] Inventory and equipment item bounding boxes sometimes return incorrectly

### ISSUE THREAD

**Issue by ghost** [powbot/issues#173]
**Describe the bug**
This bug causes the client to fail to interact with items randomly.

**To Reproduce/Line of code causing issue**
Open and close inv or equipment while drawing an item.

```
    @Subscribe
    public void onRender(RenderEvent event) {
        Item pouch = Inventory.stream().name("Colossal pouch").first();
        if(pouch.valid()) {
            Rendering.setScale(1.0f);
            Rendering.drawRect(pouch.boundingRect());
        }
    }
```

https://user-images.githubusercontent.com/64224090/205281114-9757ecd4-d4d1-4feb-9f34-1216cbe6b998.mp4

**Comment by constt** [comment]
@Protoprize was the game loading when it was wrong?

**Comment by ghost** [comment]
@constt yes, but I've noticed this happen when you're switching interfaces as well. Widget and everything else is valid, but the bounding box for it is way off sometimes. Breaks a couple of other things like alching.
